eng/pipelines/aggregate-reports.yml

```diff
@@ -49,6 +49,19 @@ stages:
               Directory: ""
               CheckLinkGuidance: $true
 
+          - task: PowerShell@2
+            displayName: "Verify generated code"
+            env:
+              EnableSourceLink: false
+            inputs:
+              filePath: "eng/scripts/CodeChecks.ps1"
+              arguments: >
+                -ServiceDirectory '*'
+                -SDKType 'all'
+                -SpellCheckPublicApiSurface
+                pwsh: true
```

Comment:
Does this have the right whitespace? It's at the same level with arguments.

Reply:
The [pipeline](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1855952&view=logs&s=f4f4e1a1-fb20-57e5-6b6d-1b695e9938d2) failed because of the alignment. I am kind of blind to check what was wrong.